Jasmine/Common/Constants.swift

```diff
@@ -78,6 +78,14 @@ enum Constants {
 
             static let tileFallInterval: TimeInterval = 0.5
         }
+
+        enum GameOver {
```

Comment:
I think this can be converted to a function that takes in the game status and spits out the text